docs/rst/notes/forthcoming_version.rst

```diff
@@ -6,6 +6,10 @@
 Forthcoming Version
 ###################
 
+Next release will include the following **major features**:
```

Comment:
(yaml validator below) Supported version v2.

Reply:
This is not correct. It supportes v3 with some failures

Reply:
To me that means not supported :)